.github/workflows/check-envvar-docs.yml

```
@@ -0,0 +1,20 @@
+name: Check env var documentation
+
+on:
+  workflow_dispatch:
+
+  push:
+
+  pull_request:
+    branches:
+      - main
+      - 'release-\d.\d\d'
```

actionlint (1.7.12)

[error] character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)